src/module/wallet/query/useGetBalanceAllAccounts.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import useServiceInstance from "../hook/useServiceInstance";
+import BigNumber from "bignumber.js";
+import useWalletState from "../hook/useWalletState";
+import ServiceInstances from "../state/ServiceInstances/ServiceInstances";
+
+export interface useGetBalanceAllAccountsProps {
+    balance: string;
+}
+
+export function useGetBalanceAllAccounts(): useGetBalanceAllAccountsProps {
+    const { state } = useWalletState();
+    const { network } = useServiceInstance();
+    const [balance, setBalance] = useState("");
+
+    useEffect(() => {
+        const getBalances = async () => {
+            const promises = state.wallets.map(async (wallet) => {
+                const serviceInstances = ServiceInstances.getServiceInstance(network, wallet.index)!;
+                const balance = await serviceInstances.getAccountBalance();
```

Comment:
const balance = await serviceInstance.getAccountBalance(); *serviceInstance sin la s

Reply:
private async getRequestsInParallel<NBR, FNR>(
        nearBlocksRequest: Promise<NBR>,
        fastNearRequest: Promise<FNR>,
    ): Promise<[NBR | undefined, FNR | undefined]> {
        //Check if `allSettled` method is supported
        if (typeof Promise.allSettled !== "function") return [await nearBlocksRequest, await fastNearRequest];

        const responseArray: [NBR | undefined, FNR | undefined] = [undefined, undefined];
        const results = await Promise.allSettled([nearBlocksRequest, fastNearRequest]);

        for (const [index, result] of results.entries()) {
            if (result.status === "fulfilled") {
                responseArray[index] = result.value;
            } else {
                // eslint-disable-next-line no-console
                console.error("Error in api service", JSON.stringify(result.reason));
            }
        }

        return responseArray;
    }